helpers/build_packages.sh

```diff
@@ -151,7 +151,60 @@ if [ "$BUILDMW" = "1" ]; then
 
     pushd $ANDROID_ROOT/hybris/mw > /dev/null
 
-    if [ "$BUILDMW_REPO" = "" ]; then
+    MANIFEST_LOOKUP=$ANDROID_ROOT
+    while true; do
+        MANIFEST=$MANIFEST_LOOKUP/.repo/manifest.xml
+        if [ -f $MANIFEST -o "MANIFEST_LOOKUP" == "$(dirname $MANIFEST_LOOKUP)" ]; then
```

Comment:
Please see: https://github.com/koalaman/shellcheck/wiki/SC2166
Test -o should be avoided. Either use [ expr ] || [ expr ] or [[ expr || expr ]].
Also  == is bash specific which I removed previously in 59a18beb5290876449f00380fcc45fa709bcd635.